CHANGELOG.rst

```diff
@@ -4,6 +4,16 @@ Ansible Utils Collection Release Notes
 
 .. contents:: Topics
 
+v2.5.2
```

Comment:
I wonder if this warrants a minor release, instead of a bugfix one. Especially, because of what this fixes and the fact that we'll be bumping netcommon's dependency on this.

@Qalthos what are your thoughts?